config/jobs/kubernetes-sigs/sig-windows/windows-op-tests.yaml

```diff
@@ -0,0 +1,69 @@
+---
+presubmits:
+  kubernetes-sigs/windows-operational-readiness:
+  - name: operational-tests-capz-windows-2019
+    decorate: true
+    always_run: false
+    optional: true
+    decoration_config:
+      timeout: 8h
+    path_alias: sig.sk8s.io/windows-operational-readiness
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220822-9f275332ba-master
+          command:
+            - runner.sh
+            - ./scripts/ci-entrypoint.sh
+          args:
+            - bash
+            - -c
+            - >-
+              cd ${GOPATH}/src/sigs.k8s.io/windows-operational-readiness/ &&
+              ./hack/run-tests.sh
+          securityContext:
+            privileged: true
+          env:
+            # CAPZ variables
+            - name: NODE_MACHINE_TYPE
+              value: "Standard_D4s_v3"
+            - name: TEST_WINDOWS
+              value: "true"
+            - name: KUBERNETES_VERSION
+              value: "v1.23.5"
```

Comment:
This should match the branch for the kubekins image we use above
If we want to test on master we can leave the container image on master and set `KUBERNETES_VERSION=latest`